components/applications-service/pkg/storage/postgres/service_groups.go

```diff
@@ -21,115 +21,74 @@ import (
-  , round((COUNT(s.health) FILTER (WHERE s.health = 'OK') / COUNT(s.health)::float) * 100) as percent_ok
-  , (
-      SELECT array_agg( DISTINCT CONCAT (s.origin, '/', s.name, '/', s.version, '/', s.release) )
-      FROM service AS s
-      WHERE s.group_id = sg.id
-    ) AS releases
-  , d.app_name as app_name
-  , d.environment as environment
-FROM service_group AS sg
-JOIN service AS s
-ON s.group_id = sg.id
-JOIN deployment as d
-ON sg.deployment_id = d.id
-GROUP BY sg.id, sg.deployment_id, sg.name, d.app_name, d.environment
-`
-	// conditional expression to calculate the overall health of the service group
-	// NOTE: @afiune We are adding a number to order the health statuses ('X_') so
-	// that we don't have to do a custom ORDER BY statement, this number must be
-	// removed before sending back any objects to the caller. (grpc functions)
-	conditionalOverallHealth = `
-CASE WHEN health_critical > 0 THEN '1_CRITICAL'
-     WHEN health_unknown  > 0 THEN '2_UNKNOWN'
-     WHEN health_warning  > 0 THEN '3_WARNING'
-     ELSE '4_OK'
-END
-`
-	// Service group health main query. Here we add the overall health calculation.
-	// NOTE: This query has NO pagination and sorting since it is reused for the HealthCounts.
-	// @afiune maybe in the future we could create a sql view so we don't have it here.
-	selectServiceGroupHealth = `
-SELECT *,(` + conditionalOverallHealth + `) health
-FROM (` + selectServiceGroupHealthCalculation + `) AS service_groups_health_calculation
-`
+	// 3) Provides an array of all service releases within a service group
+	// 4) Determines the overall status of the service group
 
 	// TODO: Update this query once we understand better the deploying status
 	selectServiceGroupsHealthCounts = `
-SELECT COUNT(*) AS total
-  , COUNT(*) FILTER (
-      WHERE health_critical > 0
-    ) AS critical
-  , COUNT(*) FILTER (
-      WHERE health_unknown  > 0
-        AND health_critical = 0
-    ) AS unknown
-  , COUNT(*) FILTER (
-      WHERE health_warning  > 0
-        AND health_critical = 0
-        AND health_unknown  = 0
-    ) AS warning
-  , COUNT(*) FILTER (
-      WHERE health_ok > 0
-        AND health_critical = 0
-        AND health_warning  = 0
-        AND health_unknown  = 0
-    ) AS ok
-FROM (` + selectServiceGroupHealth + `) AS service_groups_health_counts
+  SELECT COUNT(*) AS total
+  ,COUNT(*) FILTER (
+             WHERE health_critical > 0
+       ) AS critical
+  ,COUNT(*) FILTER (
+             WHERE health_unknown  > 0
+               AND health_critical = 0
+      ) AS unknown
+  ,COUNT(*) FILTER (
+             WHERE health_warning  > 0
+               AND health_critical = 0
+               AND health_unknown  = 0
+       ) AS warning
+  ,COUNT(*) FILTER (
+             WHERE health_ok > 0
+               AND health_critical = 0
+               AND health_warning  = 0
+               AND health_unknown  = 0
+       ) AS ok
+FROM service_groups_health AS service_groups_health_counts
```

Comment:
name still wrong. it should be singular, not plural.
```suggestion
FROM service_group_health AS service_group_health_counts
```